Package/ExcelDna.AddIn/content/ExcelDna.Build.props

```diff
@@ -0,0 +1,47 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="ExcelDnaProps">
+<!--
+  If you change properties in this file, they may not come into effect until you:
+     * Rebuild the solution/project
+
+     or
+
+     * Close Visual Studio
+     * Delete .vs folder, if exists
+     * Delete ProjectName.csproj.user (or equivalent for VB, F#, etc.), if exists
+     * Delete SolutionName.suo, if exists
+     * Open your solution/project again in Visual Studio
+  -->
+
+  <!--
+    Configuration properties for building .dna files
+  -->
+  <PropertyGroup>
+    <!--
+      Enable/Disable automatic generation of platform-specific versions of .dna files
+    -->
+    <ExcelDnaCreate32BitAddIn Condition="'$(ExcelDnaCreate32BitAddIn)' == ''">true</ExcelDnaCreate32BitAddIn>
+    <ExcelDnaCreate64BitAddIn Condition="'$(ExcelDnaCreate64BitAddIn)' == ''">true</ExcelDnaCreate64BitAddIn>
+
+    <!--
+      Define the suffix used for each platform-specific file e.g. MyAddIn64.dna
+    -->
+    <ExcelDna32BitAddInSuffix Condition="'$(ExcelDna32BitAddInSuffix)' == ''"></ExcelDna32BitAddInSuffix>
+    <ExcelDna64BitAddInSuffix Condition="'$(ExcelDna64BitAddInSuffix)' == ''">64</ExcelDna64BitAddInSuffix>
+  </PropertyGroup>
+
+  <!--
+    Configuration properties for packing .dna files
+  -->
+  <PropertyGroup>
+    <!--
+      Enable/Disable packing of .dna files
+    -->
+    <RunExcelDnaPack Condition="'$(RunExcelDnaPack)' == ''">true</RunExcelDnaPack>
+
```

Comment:
The NoCompression flag is already supported... [See the .targets file](https://github.com/Excel-DNA/ExcelDna/pull/27/files/764ce5d26c7dc272cf83c52e696dfaf376eeddda#diff-62f2797be89bc72359a89542376edfbdR117). I just didn't include it in the .props file, thinking that this is more of an advanced option.

i.e. if the developer adds the following to the .props file, it will work:

`<ExcelDnaPackCompressResources Condition="'$(ExcelDnaPackCompressResources)' == ''">false</ExcelDnaPackCompressResources>`

The same applies to the NoMultithreading packing option

`<ExcelDnaPackRunMultithreaded Condition="'$(ExcelDnaPackRunMultithreaded)' == ''">false</ExcelDnaPackRunMultithreaded>`